### DISCUSSION TITLE
Partage ressources python

### DISCUSSION THREAD

**Discussion by uggla** [meetup-python-grenoble/meetup-python-grenoble.github.io#171] [Show and tell]
@rclement , @Pierre-Loic mes liens présentés hier (2023-12-22):

- Real Python : https://realpython.com/
- Refactoring Guru : https://refactoring.guru/
- Le book pour Rust: https://doc.rust-lang.org/book/
- L'advent of code 2023, pour travailler les algos:  https://adventofcode.com/2023
- La vidéo d’Underscore l'émission de Micode sur les fameux papyrus et qui explique le challenge associé sur l'IA  : https://www.youtube.com/watch?v=W_UQHr49Tvo

**Comment by rclement** [comment]
Merci @uggla, je mettrai à jour le site cette semaine !

**Reply by rclement** [reply]
C'est en ligne: https://meetup-python-grenoble.github.io/events/2023-12-21/#liens
